plugins/webapi/value/sendtransactionbyjson/handler.go

```diff
@@ -121,6 +127,38 @@ func NewTransactionFromJSON(request Request) (*transaction.Transaction, error) {
 
 	// prepare transaction
 	tx := transaction.New(transaction.NewInputs(inputs...), transaction.NewOutputs(outputs))
+	emptyID := identity.ID{}
+
+	pledgeAccessManaNode, err := mana.IDFromStr(request.AccessMana)
+	if err != nil {
+		return nil, err
+	}
+	if pledgeAccessManaNode == emptyID {
+		pledgeAccessManaNode = local.GetInstance().ID()
+	}
+	allowedAccessMana := manaPlugin.GetAllowedPledgeNodes(mana.AccessMana)
+	if allowedAccessMana.IsFilterEnabled {
+		if !allowedAccessMana.Allowed.Has(pledgeAccessManaNode) {
+			return nil, fmt.Errorf("not allowed to pledge access mana to %s: %w", request.AccessMana, ErrNotAllowedToPledgeManaToNode)
+		}
+	}
```

Comment:
I think this only makes sense if `pledgeAccessManaNode != local.GetInstance().ID()`

Reply:
Or, preload own ID in `verifyPledgeNodes` into `allowedPledgeNodes` for all types.